src/current/_includes/releases/v24.2/v24.2.0-beta.2.md

```diff
@@ -0,0 +1,50 @@
+## v24.2.0-beta.2
+
+Release Date: July 24, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-0-beta-2-security-updates">Security updates</h3>
+
+- URLs in the following SQL statements are now sanitized of any secrets before being written to [unredacted logs]({% link v24.2/configure-logs.md %}#redact-logs). [#127360][#127360]
+- [`ALTER BACKUP SCHEDULE`]({% link v24.2/alter-backup.md %})
+- [`ALTER BACKUP`]({% link v24.2/alter-backup.md %})
+- [`ALTER CHANGEFEED SET sink`]({% link v24.2/alter-changefeed.md %})
+- [`BACKUP`]({% link v24.2/backup.md %})
+- [`COPY`]({% link v24.2/copy-from.md %})
+- [`CREATE CHANGEFEED`]({% link v24.2/create-changefeed.md %})
+- [`CREATE EXTERNAL CONNECTION`]({% link v24.2/create-external-connection.md %})
+- [`CREATE SCHEDULE FOR BACKUP`]({% link v24.2/create-schedule-for-backup.md %})
+- [`CREATE SCHEDULE FOR CHANGEFEED`]({% link v24.2/create-schedule-for-changefeed.md %})
+- [`EXPORT`]({% link v24.2/export.md %})
+- [`IMPORT`]({% link v24.2/import.md %})
+- [`RESTORE`]({% link v24.2/restore.md %})
+- [`SHOW BACKUPS`]({% link v24.2/show-backups.md %})
+- [`SHOW BACKUP`]({% link v24.2/show-backups.md %})
+
+<h3 id="v24-2-0-beta-2-sql-language-changes">SQL language changes</h3>
+
+- Added the [cluster setting]({% link v24.2/cluster-settings.md %}) [`sql.auth.grant_option_for_owner.enabled`](). The default value is `true`, which results in behavior that matches the existing behavior of CockroachDB. When set to `false`, then the [`GRANT OPTION`]({% link v24.2/show-grants.md %}#privilege-grants) is not implicitly given to the owner of an object. The object owner still implicitly has all privileges on the object, just not the ability to grant them to other [users]({% link v24.2/security-reference/authorization#roles.md %}). [#126960][#126960]
+- Fixed a bug in which the `DISCARD` statement was disallowed when the [session setting]({% link v24.2/session-variables.md %}#default-transaction-read-only) `default_transaction_read_only = on`. [#127364][#127364]
```

Comment:
```suggestion
- The [`DISCARD`]({% link v24.2/discard.md %}) parameter of [`ALTER TABLE`]({% link v24.2/alter-table.md %}#parameters) can now be used in a read-only transaction when the [session setting]({% link v24.2/session-variables.md %}#default-transaction-read-only) `default_transaction_read_only is set to `on`, This matches the behavior of PostgreSQL. [#127364][#127364] [#127364][#127364]
```

Reply:
AFAICT that's not the 'DISCARD' this PR is referring to - the test is here: https://github.com/cockroachdb/cockroach/pull/127298/commits/2c32e2008fcb4cf815dbf28b19d058facdf10a7e

it's this DISCARD which we don't document, not sure why

https://www.postgresql.org/docs/current/sql-discard.html

we have https://cockroachlabs.atlassian.net/browse/DOC-6699 and https://cockroachlabs.atlassian.net/browse/DOC-10475, one of which will require adding some docs for DISCARD, I think